bears/rest/reSTLintBear.py

```diff
@@ -14,13 +28,17 @@ class reSTLintBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting', 'Syntax'}
 
-    def run(self, filename, file):
+    def run(self, filename, file, ignore_unknown_roles: bool=False):
```

Comment:
directives are not roles, so either this is `ignore_unknown: bool`, or it is split into `ignore_unknown_roles` and `ignore_unknown_directives`.

Even if we did plan on deprecating this bear, this PR still makes sense, as it will help us implement the bear, and thus understand the bear ;-)